Makefile

```diff
@@ -1,24 +1,23 @@
 .PHONY: requirements
 
 requirements: requirements.js
-	pip install -q -r requirements/base.txt --exists-action w
+	pip install .
 
 requirements.js:
 	npm install
 
-requirements.test: requirements
-	pip install -q -r requirements/test.txt --exists-action w
-
 install: requirements
 
-develop: requirements.test
+develop: requirements
+	pip install -U pytest>=2.7 pytest-mock pycodestyle pylint edx-lint==0.5.1
 
 clean:
 	find . -name '*.pyc' -delete
```

Comment:
The crawler can output data to whatever directory you specify, so there's no way to reliably do this. I can set it to delete the "data" directory if it finds it, though, since that's the default output location.